src/translations/en.json

```
@@ -611,6 +611,9 @@
         "updater": {
           "title": "Update Instructions"
         },
+        "remote": {
+          "activity": "Current Activity"
```

```suggestion
          "activity": "Current activity"
```
In your screenshot in the background it says "Last activity" as an attribute, with a lower case "A" so this would be more consistent.